### PR TITLE
INFRA: Initialize Console App

### PR DESCRIPTION
**Pull request by GracelinJenifa**

### CODE REVIEW

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
A couple of things:
1. PR title should be 'INFRA:'
2. The commit should be 'INFRA:'
3. The project name should be 'Address.ConsoleApp'

**Comment by GracelinJenifa** [comment]
> A couple of things:
> 
> 1. PR title should be 'INFRA:'
> 2. The commit should be 'INFRA:'
> 3. The project name should be 'Address.ConsoleApp'

@ShriHumrudha  I have made the my changes to the PR. Kindly check.

File: Address ConsoleApp/Address. ConsoleApp.csproj [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (ShriHumrudha) [inline Address ConsoleApp/Address. ConsoleApp.csproj]: Till it is the same
image

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by GracelinJenifa** [comment]
@ShriHumrudha  I have Pushed a new PR #3  for this task, so closing this.

**Pull request closed without merging**